mm/2s2h/GameInteractor/GameInteractor.h

```diff
@@ -216,6 +216,7 @@ typedef enum {
     VB_GIBDO_TRADE_SEQUENCE_TAKE_MORE_THAN_ONE_ITEM,
     VB_GIBDO_TRADE_SEQUENCE_DO_TRADE,
     VB_GET_ITEM_ACTION_FROM_MASK,
+    VB_GRASS_STORE_RANDO_CHECK_ID,
```

Comment:
This VB_ name is not really a good name to reflect the behavior of code that it is wrapping.